### PR TITLE
Improve ranking battle reward

### PR DESCRIPTION
**Pull request by ipdae**
- Drop consumable materials when PVP finished.

### CODE REVIEW

File: .Lib9c.Tests/Action/RankingBattleTest.cs [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    public class RankingBattleTest : IDisposable
+    {
+        private TableSheetsState _tableSheetsState;
+
+        public RankingBattleTest()
+        {
+            _tableSheetsState = TableSheetsImporter.ImportTableSheets();
+        }
+
+        public void Dispose()
```
Comment (longfin) [inline .Lib9c.Tests/Action/RankingBattleTest.cs:25]: I guess it's okay that we skip this `Dispose()`. (`null` assigning doesn't affect other test cases.)

File: .Lib9c.Tests/Lib9c.Tests.csproj [hunk 2]
```diff
@@ -28,6 +28,12 @@
 
   <ItemGroup>
     <Content Include="Data/**/*.*" CopyToOutputDirectory="Always" />
+    <Content Update="Data\WeeklyRewardSheet.csv">
```
Comment (longfin) [inline .Lib9c.Tests/Lib9c.Tests.csproj:31]: Are these `<Content>`s needed?

**Comment by ipdae** [comment]
mic check

File: Lib9c/TableData/TableSheets.cs [hunk 3]
```diff
@@ -109,7 +110,15 @@ public static TableSheets FromTableSheetsState(TableSheetsState tableSheetsState
             var tableSheets = new TableSheets();
             tableSheets.InitializeWithTableSheetsState(tableSheetsState);
 
-            Cache.Add(tableSheetsState, tableSheets);
+            // Avoid Exception when run parallel tests
```
Comment (longfin) [inline Lib9c/TableData/TableSheets.cs:113]: It seems that we should reconsider the current table sheets caching strategy 😢

**Pull request merged**